### PR TITLE
p2p crawler add api

### PR DESCRIPTION
**Pull request by bCoder778**

### CODE REVIEW

**Review by jamesvan2019 — APPROVED** [review]

File: cmd/crawler/config/config.go [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+	}
+
+	if numNets == 0 {
+		numNets++
+		params.ActiveNetParams = &params.MainNetParam
+	}
+
+	if len(c.BootstrapNodeAddr) == 0 {
+		c.BootstrapNodeAddr = params.ActiveNetParams.Bootstrap
+		fmt.Println(c.BootstrapNodeAddr)
```
Comment (lochjin) [inline cmd/crawler/config/config.go:176]: We should use `log` instead

File: cmd/crawler/config/config.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
+package config
```
Comment (lochjin) [inline cmd/crawler/config/config.go:1]: It's better to merge with https://github.com/bCoder778/qitmeer/blob/0.10-dev/cmd/crawler/node/config.go

**Review by lochjin — CHANGES_REQUESTED** [review]

File: cmd/crawler/config/config.go (hunk 1 above)
Comment (bCoder778) [reply cmd/crawler/config/config.go:176]: ok

File: cmd/crawler/config/config.go (hunk 2 above)
Comment (bCoder778) [reply cmd/crawler/config/config.go:1]: ok

**Review by lochjin — APPROVED** [review]

**Pull request merged**